Gemfile

```diff
@@ -1,15 +1,16 @@
 source 'https://rubygems.org'
 gemspec
 
+gem 'openssl-ccm', github: 'adfoster-r7/openssl-ccm', branch: 'add-support-openssl-3'
+gem 'openssl-cmac', github: 'adfoster-r7/openssl-cmac', branch: 'add-support-for-openssl3'
```

Comment:
We can't ship this RubySMB PR until these gems are bumped

Reply:
These were shipped:

- https://github.com/SmallLars/openssl-ccm/pull/10
- https://github.com/SmallLars/openssl-cmac/pull/5